src/commonMain/kotlin/com/monta/slack/notifier/SlackClient.kt

```diff
@@ -6,9 +6,9 @@ import com.monta.slack.notifier.util.client
 import io.ktor.client.request.*
 import io.ktor.client.statement.*
 import io.ktor.http.*
+import io.ktor.utils.io.charsets.*
```

Comment:
nitpick: welllllll, now that reviewdog is enabled, it might be time to update the settings in intellij to not wildcard import 😝

Reply:
You got it :-) - I added ktlint and autocorrected.
`io.ktor` we have in general as wildcard imports though, so I will keep those.
I have these settings
![image](https://github.com/monta-app/slack-notifier-cli/assets/5623209/26884a90-15a3-439b-bc27-2e4f9b3ca7d6)